server/delay_server/src/irodsDelayServer.cpp

```diff
@@ -676,6 +677,51 @@ namespace
     } // make_delay_queue_query_processor
 } // anonymous namespace
 
+bool is_current_server_leader(void) {
+    std::string hostname;
+    char hn[HOST_NAME_MAX + 1]{};
+
+    if (gethostname(hn, sizeof(hn)) != 0) {
+        logger::delay_server::warn("is_current_server_leader: Failed to retrieve local server's hostname.");
```

Comment:
You can use `__func__` as an argument to avoid having to manually type the function name.
```cpp
logger::delay_server::warn("{}: Failed to retrieve local server's hostname.", __func__);
```

Reply:
Oops, knew about `__func__` but didn't know the logging was formatted. Fixing!